### DISCUSSION TITLE
pinia数据持久化

### DISCUSSION THREAD

**Discussion by l1548844035** [vbenjs/vue-vben-admin#2895] [General]
这个项目是怎么做到pinia的数据持久化的，就是页面刷新store的数据还在那种

**Comment by likui628** [comment]
getters里读取缓存
https://github.com/vbenjs/vue-vben-admin/blob/e31525a803ee5a7c9770b4eccebe91cc90f7aa60/src/store/modules/user.ts#L46-L48
actions里设置缓存
https://github.com/vbenjs/vue-vben-admin/blob/e31525a803ee5a7c9770b4eccebe91cc90f7aa60/src/store/modules/user.ts#L60-L63
